CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPane.java

```diff
@@ -1033,6 +1036,23 @@ private void deselectAllPlugins() {
         });
     }
 
+    @Override
+    public void qualityControlRuleChanged(final boolean canRun) {
+        if (canRun) {
```

Comment:
The "stop" style is also used when a query is running - i'd probably either create a new "thinking" style or just disable the button when `canRun = false` to avoid any confusion.